tmc-langs-framework/src/test/java/fi/helsinki/cs/tmc/langs/ClassPathTest.java

```diff
@@ -0,0 +1,74 @@
+
+package fi.helsinki.cs.tmc.langs;
+
+import com.google.common.base.Throwables;
+import java.io.File;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ClassPathTest {
+    
+    private ClassPath cp;
+    
+    @Before
+    public void setUp() {
+        cp = new ClassPath(getPath("ant_project"));
```

Comment:
Requiring at least one path in the constructor seems unnecessary and possibly makes the interface troublesome in some use cases (which you don't have yet). Is there a reason for this?

Maybe taking a variadic number (`ClassPath(Path... paths)`) would be handy.